src/main/java/org/apache/jmeter/protocol/aws/cognito/CognitoProducerAdminCreateUser.java

```diff
@@ -0,0 +1,111 @@
+package org.apache.jmeter.protocol.aws.cognito;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.jmeter.config.Argument;
+import org.apache.jmeter.config.Arguments;
+import org.apache.jmeter.protocol.java.sampler.JavaSamplerContext;
+import org.apache.jmeter.samplers.SampleResult;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
When removing the JsonProcessingException statements then you can remove unused imports.